src/InventoryManager.cpp

```diff
@@ -78,3 +79,68 @@ int InventoryManager::userInput()
     
 	return 0;
 }
+
+/*uses ActiveInventory functions to create items from a csv file*/
+void InventoryManager::readCSVFile(const std::string &file)
+{
+	std::string name, str_id, cat, sub_cat, qty, sale_price;
+	std::string tax, total_price, buy_cost, profit, exp, tmp_line; 
+	unsigned long id;
+	
+	std::ifstream csv_file(file);
+
+	if(!csv_file.is_open()) std::cout << "ERROR: File Open" << '\n';
```

Comment:
This error message kind of makes it sound like the file is open and that's an error.  Maybe change this error message to something like:
```ERROR: Unable to open file.```